### ISSUE TITLE
Write a migration guide (from other simulator)

### ISSUE THREAD

**Issue by lukicdarkoo** [cyberbotics/webots_ros2#199]
Here is a very nice migration guide example:
https://community.gazebosim.org/t/migration-from-gazebo-classic-to-ignition-gazebo/805
Besides the migration documentation, they also have case-studies.

I believe we should make something similar.

**Comment by omichel** [comment]
Yes, we should add such a migration guide to the [wiki](https://github.com/cyberbotics/webots_ros2/wiki) pages.
